test/services/VatSubscriptionServiceSpec.scala

```diff
@@ -206,4 +206,29 @@ class VatSubscriptionServiceSpec extends TestUtil with MockVatSubscriptionConnec
       result shouldBe expectedPPOB
     }
   }
+
+  "calling updatePhoneNumbers" when {
```

Comment:
Could you also add some unit tests for `buildPhoneNumbersUpdateModel`

Reply:
Good point we missed that